src/NuGet.Clients/NuGet.PackageManagement.UI/Models/PackageSolutionDetailControlModel.cs

```diff
@@ -127,30 +143,35 @@ private async Task UpdateInstalledVersionsAsync(CancellationToken cancellationTo
             {
                 try
                 {
-                    IPackageReferenceContextInfo installedVersion = await GetInstalledPackageAsync(project.NuGetProject, Id, cancellationToken);
-                    if (installedVersion != null)
+                    (IPackageReferenceContextInfo TopLevelPackage, IPackageReferenceContextInfo TransitivePackage) installedPackage = await GetInstalledPackageAsync(project.NuGetProject, Id, cancellationToken);
+                    var topLevelPackageVersion = installedPackage.TopLevelPackage;
+                    var transitivePackageVersion = installedPackage.TransitivePackage;
+
+                    project.RequestedVersion = null;
+                    project.InstalledVersion = null;
+                    project.PackageLevel = null;
+                    project.AutoReferenced = false;
+                    project.InstalledVersionMaxVulnerability = -1;
```

Comment:
Nit: I don't see a lot of gain from moving this out of the else case. It does set some defaults, but they get overridden in the 2 specified cases, so it would be fine to leave them in the else case.